distributed/joblib.py

```diff
@@ -196,20 +196,21 @@ def maybe_to_futures(args):
                     continue
 
                 f = None
-                if f is None and arg_id in self.data_futures:
+                if arg_id in self.data_futures:
                     f = self.data_futures[arg_id]
 
                 elif f is None and call_data_futures is not None:
                     try:
                         f = call_data_futures[arg]
                     except KeyError:
-                        if is_weakrefable(arg) and sizeof(arg) > 1e6:
-                            # Automatically scatter large objects to some of
-                            # the workers to avoid duplicated data transfers.
-                            # Rely on automated inter-worker data stealing if
-                            # more workers need to reuse this data concurrently
-                            # beyond the initial broadcast arity.
-                            [f] = self.client.scatter([arg], broadcast=3)
+                        if is_weakrefable(arg) and sizeof(arg) > 1e3:
+                            # Automatically scatter large objects to one of the
+                            # workers to avoid duplicated data transfers. Rely
+                            # on automated inter-worker data stealing if more
+                            # workers need to reuse this data concurrently as
+                            # well as the ability of a worker to refuse data
+                            # requests with busy signal.
+                            [f] = self.client.scatter([arg], broadcast=1)
```

Comment:
You can just remove the `broadcast=` keyword here.

Reply:
Explicit is better than implicit, no?

Reply:
If you like.  No strong thoughts.  One argument against is we tend not to explicitly state defaults.  Being fully explicit and listing all keywords would be verbose.  There is history here though so listing this keyword might be a good idea.  Again, no strong thoughts.